### ISSUE TITLE
[bug] NUnit2046 doesn't flag Enumerable.SequenceShouldEqual(Enumerable)

### ISSUE THREAD

**Issue by Bartleby2718** [nunit/nunit.analyzers#750]
Very similar to #748, but for [`.SequenceEqual()`](https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.sequenceequal). NUnit.Analyzers should be able to convert
```cs
var expected = new[] { 1, 2 };
var actual = new[] { 1, 2 };
Assert.That(actual.SequenceEqual(expected));
```
to
```cs
Assert.That(actual, Is.EquivalentTo(expected));
```
but it doesn't.

**Comment by Bartleby2718** [comment]
@manfred-brands Do you think this should be part of [NUnit2046](https://docs.nunit.org/articles/nunit-analyzers/NUnit2046.html) or not?

[The corresponding analyzer `UseCollectionConstraintAnalyzer.cs`](https://github.com/nunit/nunit.analyzers/blob/master/src/nunit.analyzers/UseCollectionConstraint/UseCollectionConstraintAnalyzer.cs) has a very generic name, but it just checks for member access operation.
